### ISSUE TITLE
Unable to install version 2.1.0-beta

### ISSUE THREAD

**Issue by AlexanderEggers** [segmentio/analytics-react-native#426]
When trying to install the latest preview version 2.1.0-beta, I get the following error:

> An unexpected error occurred: "could not find a copy of react-native to link in /Users/alex/Documents/Projects/mobile-app/node_modules/@segment/analytics-react-native/node_modules".

- `analytics-react-native` version: 2.1.0-beta
- React Native version: 0.67.1

**Steps to reproduce**

Add 2.1.0-beta to your RN project's package.json and run `yarn install`.

**Expected behavior**

Latest preview can be installed without issues.

**Actual behavior**

> An unexpected error occurred: "could not find a copy of react-native to link in /Users/alex/Documents/Projects/mobile-app/node_modules/@segment/analytics-react-native/node_modules".

**Comment by Kerumen** [comment]
I'm also unable to install the latest version but with a different error:

```
error An unexpected error occurred: "expected hoisted manifest for \"@segment/analytics-react-native#react-native#metro-babel-register\"".
```

**Comment by alanjcharles** [comment]
Hi @AlexanderEggers @Kerumen apologies- it seems we have a dependency issue that didn't present itself in our tests. Can you both try to add `@segment/sovran-react-native` before adding the `@segment/analytics-react-native@2.1.0`? I am able to get around this issue with this strategy. We will push an update as soon as we can to resolve this. Thanks so much for reporting.

**Comment by AlexanderEggers** [comment]
@alanjcharles I tried to add @segment/sovran-react-native, but that is resulting in build errors for ios. I will try to provide some more details in a while.

**Comment by alanjcharles** [comment]
@AlexanderEggers @Kerumen resolved in `@segment/analytics-react-native@2.1.1-beta` please upgrade and feel free to reopen if issue persists. Thanks again

**Comment by AlexanderEggers** [comment]
@alanjcharles New version is working fine for me. Thanks for quickly publishing a new release 🙌
